DependencyInjection/SonataFormatterExtension.php

```diff
@@ -56,8 +56,13 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('ckeditor.xml');
         }
 
-        // TODO: To be removed when major version is changed
+        // NEXT_MAJOR: remove this if block
         if (!isset($config['default_formatter'])) {
+            @trigger_error(
+                'Not setting the default_formatter configuration node is deprecated since 3.x,'.
```

Comment:
I wouldn't split this string in two parts just for the character limit, because searching for this particular string in the whole projects wouldn't work later. 

This is a general discussion I don't want to start :trollface: